@app/lib/src/GraphileApolloLink.ts

```diff
@@ -53,22 +52,32 @@ export class GraphileApolloLink extends ApolloLink {
             }
             return;
           }
-          const schema = await postgraphileMiddleware.getGraphQLSchema();
-          const data =
-            await postgraphileMiddleware.withPostGraphileContextFromReqRes(
-              req,
-              res,
-              {},
-              (context) =>
-                execute(
-                  schema,
-                  document,
-                  rootValue || {},
-                  context,
-                  variableValues,
-                  operationName
-                )
-            );
+          const schema = await pgl.getSchema();
+          const args = {
+            schema,
+            document,
+            variableValues,
+            operationName,
+          };
+          await hookArgs(
+            args,
+            {
+              node: {
+                req,
+                res,
+              },
+              expressv4: {
+                req,
+                res,
+              },
+            },
+            pgl.getResolvedPreset()
```

Comment:
`hookArgs` is how the context is derived now, it means `withPostGraphileContextFromReqRes` is no longer needed. We pass the `req`/`res` objects (both into `expressv4` and `node` scopes) so that `hookArgs` can use them to call your `context` callback (or `pgSettings`/`additionalGraphQLContextFromRequest` if you're using the V4 preset, which we are) to derive the GraphQL context.